README.md

```diff
@@ -139,7 +146,7 @@ shape: (5, 8)
 ### Blazingly fast
 
 Polars is very fast. In fact, it is one of the best performing solutions available.
-See the results in [h2oai's db-benchmark](https://h2oai.github.io/db-benchmark/).
+See the results in [DuckDB's db-benchmark](https://duckdblabs.github.io/db-benchmark/).
```

Comment:
I think the direct link is much more readable: https://tmonster.github.io/h2oai-db-benchmark/

Reply:
They render exactly the same in my browser, though?

(To clarify: I'm linking directly to the [benchmark itself](https://duckdblabs.github.io/db-benchmark/) rather than the [blog post](https://duckdb.org/2023/04/14/h2oai.html) with embedded timings.)

Reply:
Oh, right. Never mind.